kolibri/plugins/learn/assets/src/learn_module.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+var _ = require('lodash');
+var logging = require('loglevel');
+
+// This is aliased for your convenience!
+var KolibriModule = require('kolibri_module');
+
+var LearnModule = KolibriModule.extend({
+
+    start: function(plugin) {
+        // This is called on the kolibri_register event - make sure it is this plugin that has registered.
```

Comment:
Initialize is explicitly called *before* registration with the Mediator - so initialize is only for things that you want to happen during the constructor function, before it is registered as a plugin.

Yes, I should list the event APIs of every KolibriModule, and the Core app. Will think on how to best present this.